### PR TITLE
fixing fetchFromInstance

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14225

## Description
Handling `invalid character \'[a-zA-Z]\' looking for beginning of value` exception

### CODE REVIEW

File: Scripts/script-FetchFromInstance.yml [hunk 1]
```diff
@@ -17,6 +19,8 @@ script: |
           contents = inc.get('Contents', '')
           if 'Error' in contents:
               raise Exception("Error occurred while fetching incidents from {0}: {1}".format(instance_name, contents))
+          if re.match("invalid character \'[a-zA-Z]\' looking for beginning of value", contents, re.IGNORECASE):
```
Comment (Itay4) [inline Scripts/script-FetchFromInstance.yml]: Do we get this error only with letter characters ?

Comment (ronykoz) [reply Scripts/script-FetchFromInstance.yml]: As far as I can recall, we can always add a fix for that

Comment (ronykoz) [reply Scripts/script-FetchFromInstance.yml]: As far as I can recall, we can always add a fix for that

File: Scripts/script-FetchFromInstance.yml [hunk 2]
```diff
@@ -17,6 +19,8 @@ script: |
...
+              raise Exception("Error occurred while fetching incidents from {0}: {1}".format(instance_name, contents))
```
Comment (Itay4) [inline Scripts/script-FetchFromInstance.yml]: please use `return_error`

File: Scripts/script-FetchFromInstance.yml [hunk 3]
```diff
@@ -1,9 +1,11 @@
 commonfields:
   id: FetchFromInstance
   version: -1
-rolename: []
 name: FetchFromInstance
+fromversion: 4.0
```
Comment (Itay4) [inline Scripts/script-FetchFromInstance.yml]: please change to `4.0.0`

**Pull request merged**